src/static/Services.js

```diff
@@ -0,0 +1,26 @@
+
+export const API = {
+    fetchAllLabs:  async ()  => {
+        const endpointAPI = "https://match-labs-api.herokuapp.com/api/no_auth/labs";
```

Comment:
To prevent duplication, you could refactor this `API` service by:
- extracting the API host in a single constant (`"https://match-labs-api.herokuapp.com/api/no_auth/labs"`)
- extracting the common code in a generic `fetchJson(resource, id)` method

Reply:
Good point, thanks!